library/src/main/java/com/novoda/downloadmanager/notifications/NotificationDisplayer.java

```diff
@@ -123,7 +123,7 @@ private void buildActionIntents(int type, Collection<DownloadBatch> cluster, Not
         }
 
         Intent clickIntent = createClickIntent(batchId, batchStatus);
-        builder.setContentIntent(PendingIntent.getBroadcast(context, 0, clickIntent, PendingIntent.FLAG_UPDATE_CURRENT));
+        builder.setContentIntent(PendingIntent.getBroadcast(context, 0, clickIntent, PendingIntent.FLAG_CANCEL_CURRENT));
```

Comment:
this is the pending intent that gets fired when the user clicks on a notification